src/go/vtunnel/test/e2e/connectivity_test.go

```diff
@@ -0,0 +1,220 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	wslTarballName     = "distro-0.21.tar"
+	wslTarballURL      = "https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/releases/download/v0.21/distro-0.21.tar"
+	wslDistroName      = "vtunnel-e2e-test"
+	testHTTPServerAddr = "127.0.0.1:3434"
+	handShakePort      = "9090"
+	vSockHostPort      = "8989"
+	peerTCPPort        = "3030"
+)
+
+func TestConnect(t *testing.T) {
+	tmpDir := t.TempDir()
+
+	t.Log("Building vtunnel host binary")
+	err := buildBinaries("../../main.go", "windows", tmpDir)
+	require.NoError(t, err, "Failed building vtunnel.exe")
+
+	t.Log("Building vtunnel peer binary")
+	err = buildBinaries("../../main.go", "linux", tmpDir)
+	require.NoError(t, err, "Failed building vtunnel")
+
+	t.Logf("Dowloading %v wsl distro tarball", wslTarballName)
```

Comment:
This is in `rancher-desktop`; we can totally just look at `resources/win32/distro.tar` or whatever it is?

Reply:
The distro name is inconsistent here, so it's just easier to always download it, at least for now.